src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalRent = 40 * days;
+  const smallDiscount = 20;
+  const largeDiscount = 50;
+
+  if (days >= 1 && days < 3) {
```

Comment:
Since the `days` parameter should always be greater than or equal to 1, there's no need to check for `days >= 1`. You can simplify this condition to `days < 3`. 🧹